code/modules/mob/living/carbon/emote.dm

```diff
@@ -34,6 +34,18 @@
 							'sound/misc/clap3.ogg',
 							'sound/misc/clap4.ogg')
 
+/datum/emote/living/carbon/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles."
+	sound = 'sound/misc/knuckles.ogg'
+	cooldown = 6 SECONDS
+
+/datum/emote/living/carbon/crack/can_run_emote(mob/living/carbon/user, status_check = TRUE , intentional)
+	if(user.get_num_arms() <= 1)
```

Comment:
This code is ugly
```suggestion
	if(user.get_num_arms() < 2)
```

Reply:
you can literally pop with one hand

Reply:
The message is cracks their knuckles with an s, stop being a cunt